madushag/MyFarm
# This PR is for the user story MYF-5 and MYF-8

### What is included in this pull request**
* Adds the ability to view a list of farms, update an existing farm, and add a new farm
* Added custom styling via Bootstrap

### How to test
I had trouble getting the postgresql instance to work via docker, so this version was only tested as a local instance against the default sqllite db instance.

* Launch the application on your local machine via manage.py runserver
* Go to localhost:8000/farm in your browser, and you should see a list of available farms in the local db

### Remaining work
* Additional form validation needed
* Need to add a few more links so there are no dead-ends to the user experience
* Need to add the ability to associate the farms with a farmer. This needs discussion as the farmer entity should be linked create and linked during the user registration process. I have created a FarmerStub model in this PR as a placeholder, but this needs to be flushed out further with the group.
* Tests need to be added
